tests/test_LogOutputHelper.cpp

```diff
@@ -127,35 +124,35 @@ BOOST_AUTO_TEST_CASE(Cumulative)
     constexpr auto fields = std::array {
         std::pair{"FOPT", 1.0},
         std::pair{"FWPT", 2.0},
-        std::pair{"FGPT", 3.0},
+        std::pair{"FGPT", 3.0e3},
```

Comment:
> Please add a comment that the "e3" is added to get just "3.0" in the output (I might have expected the test to change the expected output instead, but this approach is fine).

I've pushed an update to include a comment to this effect.  I decided to change the input instead of the output, because the small cumulative gas values would otherwise all be output as `0.0` in `MM*` units with the precision here.